src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/_help.py

```diff
@@ -0,0 +1,206 @@
+          text: |-
+            az bot telegram create -n botName -g MyResourceGroup --access-token token
+            --is-validated
+"""
+helps['bot sms create'] = """
+    type: command
+    short-summary: Create an SMS Channel on a bot.
+    examples:
+        - name: Create an SMS Channel for a bot.
+          text: |-
+            az bot sms create -n botName -g MyResourceGroup --account-sid sid
+            --auth-token token --is-validated
+"""
+helps['bot slack create'] = """
+    type: command
+    short-summary: Create a Slack Channel on a bot.
+    examples:
+        - name: Create a Slack Channel for a bot.
+          text: |-
+            az bot slack create -n botName -g MyResourceGroup --client-id clientid
+            --client-secret secret --verification-token token
+"""
+helps['bot authsetting'] = """
+    type: group
+    short-summary: Manage OAuth Connection Settings on a bot.
+"""
+helps['bot authsetting create'] = """
+    type: command
+    short-summary: Create an OAuth Connection Setting on a bot.
+    examples:
+        - name: Create a new OAuth Connection Setting on a bot.
+          text: |-
+            az bot authsetting create -g MyResourceGroup -n botName -c myConnectionName
+            --client-id clientId --client-secret secret --scopes "scope1 scope2" --service google
+            --parameters id=myid
+"""
+helps['bot authsetting show'] = """
+    type: command
+    short-summary: Show details of an OAuth Connection Setting on a bot.
+"""
+helps['bot authsetting list'] = """
+    type: command
+    short-summary: Show all OAuth Connection Settings on a bot.
+"""
+helps['bot authsetting delete'] = """
+    type: command
+    short-summary: Delete an OAuth Connection Setting on a bot.
+"""
+helps['bot authsetting list-providers'] = """
+    type: command
+    short-summary: List Details of All service Providers available for creating OAuth Connection Settings.
+    examples:
+        - name: List all service providers.
+          text: |-
+            az bot authsetting list-providers
+        - name: Filter by a particular type of service provider.
+          text: |-
+            az bot authsetting list-providers --provider-name google
+"""
+
+
+for channel in ['Facebook', 'email', 'Skype', 'Kik', 'Directline', 'Telegram', 'SMS', 'Slack']:
+    channel_name = channel.lower()
+    helps['bot {0} delete'.format(channel_name)] = """
+        type: command
+        short-summary: Delete the {0} Channel on a bot
+    """.format(channel)
+    helps['bot {0} show'.format(channel_name)] = """
+        type: command
+        short-summary: Get details of the {0} Channel on a bot
+    """.format(channel)
+    helps['bot {0}'.format(channel_name)] = """
+        type: group
+        short-summary: Manage the {0} Channel on a bot.
+    """.format(channel)
+
+
+helps['bot msteams delete'] = """
+    type: command
+    short-summary: Delete the Microsoft Teams Channel on a bot
```

Comment:
Incorrect article; `the` -> `a`. Unless there is only allowed to be _one_ such channel on a bot, in which case other documentation may need to be edited. Applies throughout.

Reply:
yes, there is only one channel of each type  allowed per bot. i am changing the article to `the` instead of `a[n]`